lib/Doctrine/DBAL/Schema/Column.php

```diff
@@ -118,9 +119,17 @@ public function setOptions(array $options)
     {
         foreach ($options as $name => $value) {
             $method = "set".$name;
-            if (method_exists($this, $method)) {
-                $this->$method($value);
+            if ( ! method_exists($this, $method)) {
+                // next major: throw an exception
+                @trigger_error(sprintf(
+                    'The "%s" option is not supported,'.
+                    ' setting it is deprecated and will cause an exception in 3.0',
```

Comment:
Probably should not mention _exception_ directly (implementation detail), just _error_.